package.json

```diff
@@ -24,6 +24,7 @@
 		"bootstrap": "^4.2.1",
 		"eslint": "^5.9.0",
 		"highlight.js": "^9.12.0",
+		"js-base64": "^2.5.1",
```

Comment:
Why do you need this dependency? There are no occurrences of it below.

Reply:
only 4 cp to assets, mb later all js assets will be generated by webpack. but now its redundantly, yes